package.json

```diff
@@ -16,13 +16,17 @@
   "dependencies": {
     "babel-preset-es2015": "^6.1.2",
     "canvas": "^1.3.0",
-    "docopt": "^0.6.2"
+    "concat-stream": "^1.5.1",
+    "docopt": "^0.6.2",
+    "duplexer": "^0.1.1",
+    "multipipe": "^0.2.1",
+    "readable-stream": "^2.0.4"
   },
   "scripts": {
     "start": "npm run dist && npm run test",
-    "dist": "babel ./lib --out-dir ./dist",
-    "lint": "semistandard",
-    "test": "rm -f ./images/sample.lego.png && ./bin/node-legofy ./images/sample.jpg"
+    "dist": "babel lib --out-dir dist",
+    "lint": "standard",
+    "test": "bin/node-legofy < images/sample.jpg > images/sample.lego.jpg"
```

Comment:
This is the Unix API, and this is the simpler possible (no arguments, just input and output!).

But yes I agree having a variant with `<input> [<output>]` is nice to have, that's why I didn't remove `docopt`.